.github/workflows/module-cleanup.yml

```diff
@@ -0,0 +1,94 @@
+name: Clean-up AWS accounts
+
+# Reusable workflow
+on:
+  workflow_call:
+    inputs:
+      modules:
+        required: true
+        type: string
+      regions:
+        required: true
+        type: string
+      global:
+        required: true
+        type: boolean
+
+jobs:
+  module-cleanup:
+    runs-on: ubuntu-latest
+    name: ${{ matrix.module }} ${{ matrix.region }}
+    strategy:
+      fail-fast: false
+      matrix:
+        module: ${{ fromJson(inputs.modules) }}
+        region: ${{ fromJson(inputs.regions) }}
+
+    env:
+      MATRIX_AWS_ACCESS_KEY_ID: AWS_ACCESS_KEY_ID_${{matrix.module}}
+      MATRIX_AWS_SECRET_ACCESS_KEY: AWS_SECRET_ACCESS_KEY_${{matrix.module}}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Start local iasql engine
+        run: |
+          IASQL_ENV=ci docker-compose up --build --detach
+          while ! curl --output /dev/null --silent --head --fail http://localhost:8088/health; do sleep 1 && echo -n .; done
+
+      - name: Create database
+        run: |
+          echo "\nCreating iasql db..."
+          curl http://localhost:8088/v1/db/connect/iasql
+
+      - name: Set up iasql database
+        run: |
+          echo "\nInstalling aws_account..."
+          psql postgres://postgres:test@localhost:5432/iasql -c "
+            select iasql_install(
+              'aws_account'
+            );
+          ";
+          echo "\Attaching credentials..."
+          psql postgres://postgres:test@localhost:5432/iasql -c "
+            INSERT INTO aws_account (region, access_key_id, secret_access_key)
+            VALUES ('${AWS_REGION}', '${AWS_ACCESS_KEY_ID}', '${AWS_SECRET_ACCESS_KEY}');
+          ";
+          echo "\Install all modules..."
+          psql postgres://postgres:test@localhost:5432/iasql -c "
+            select iasql_module_install(
+              array(select module_name from iasql_modules_list() where module_name != 'aws_account' and module_name not like 'iasql_%')
+            );
+          ";
+        env:
+          AWS_REGION: ${{ matrix.region }}
+          AWS_ACCESS_KEY_ID: ${{ secrets[env.MATRIX_AWS_ACCESS_KEY_ID] }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets[env.MATRIX_AWS_SECRET_ACCESS_KEY] }}
+
+      - name: Run cleaning script
+        if: ${{ !inputs.global }}
+        run: |
+          echo "\nDelete all records..."
+          psql postgres://postgres:test@localhost:5432/iasql -f test/sql/delete_records.sql -c "select * from iasql_apply();"
```

Comment:
Minor question: Why not shove the `select * from iasql_apply()` into the `.sql` files?

Reply:
That works for me. :)